tests/e2e/Services/Databases/DatabasesCustomServerTest.php

```diff
@@ -593,6 +593,200 @@ public function testUpdateCollection(array $data)
+                'alias' => 'JJ'
+            ],
+            'permissions' => [
+                Permission::read(Role::any()),
+                Permission::update(Role::any()),
+                Permission::delete(Role::any()),
+            ],
+        ]);
+
+        $index = $this->client->call(Client::METHOD_POST, '/databases/' . $databaseId . '/collections/' . $actors['body']['$id'] . '/indexes', array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+            'x-appwrite-key' => $this->getProject()['apiKey']
+        ]), [
+            'key' => 'key_lastName',
+            'type' => 'key',
+            'attributes' => [
+                'lastName',
+            ],
+        ]);
+
+        // Wait for database worker to finish creating index
+        sleep(2);
+
+        $collection = $this->client->call(Client::METHOD_GET, '/databases/' . $databaseId . '/collections/' . $actors['body']['$id'], array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+            'x-appwrite-key' => $this->getProject()['apiKey']
+        ]), []);
+
+        $this->assertEquals(200, $collection['headers']['status-code']);
+        $this->assertIsArray($collection['body']['attributes']);
+        $this->assertCount(4, $collection['body']['attributes']);
+        $this->assertEquals($collection['body']['attributes'][0]['key'], $firstName['body']['key']);
+        $this->assertEquals($collection['body']['attributes'][1]['key'], $lastName['body']['key']);
+        $this->assertEquals($collection['body']['attributes'][2]['key'], $age['body']['key']);
+        $this->assertEquals($collection['body']['attributes'][3]['key'], $alias['body']['key']);
+        $this->assertCount(1, $collection['body']['indexes']);
+        $this->assertEquals($collection['body']['indexes'][0]['key'], $index['body']['key']);
+
+        $unneededId = $alias['body']['key'];
+
+        // Delete attribute
+        $attribute = $this->client->call(Client::METHOD_DELETE, '/databases/' . $databaseId . '/collections/' . $actors ['body']['$id'] . '/attributes/' . $unneededId, array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+            'x-appwrite-key' => $this->getProject()['apiKey']
+        ]));
+
+        $this->assertEquals(204, $attribute['headers']['status-code']);
+
+        sleep(2);
+
+        // Check document to ensure cache is purged on schema change
+        $document = $this->client->call(Client::METHOD_GET, '/databases/' . $databaseId . '/collections/' . $actors['body']['$id'] . '/documents/' . $document['body']['$id'], array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+            'x-appwrite-key' => $this->getProject()['apiKey']
+        ]));
+
```

Comment:
Let's add an assertion to check that `lastName` is decrypted properly when getting the document